.github/workflows/centos.yml

```diff
@@ -172,5 +153,25 @@ jobs:
           cd build
           ctest --output-on-failure -C Release -R cxx_cpp -E 'cxx_cpp_linear_programming'
 
+      - name: Upload OR-Tools install artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ steps.or-install.outputs.archive_name }}
+          path: ${{ steps.or-install.outputs.archive_path }}
 
+  publish_asset:
+    name: Publish release assets
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download artifacts
+        if: ${{ env.RELEASE_CREATED == 'true' }}
+        uses: actions/download-artifact@v3
 
+      - name: Publish assets
+        if:  ${{ env.RELEASE_CREATED == 'true' }}
+        uses: alexellis/upload-assets@0.4.0
```

Comment:
why can't we use the official action `actions/upload-release-asset` ?

Reply:
Because this action is no longer maintained since 2021, see [upload-release-asset](https://github.com/actions/upload-release-asset).

Reply:
Yes, sorry, I noticed.
I've seen that the proposed alternative is
https://github.com/softprops/action-gh-release
If the latter does not seem appropriate I'm ok with merging the this PR

Reply:
It seems that [action-gh-release](https://github.com/softprops/action-gh-release) doesn't accept wildcards (_build/*.zip).

It also seems that it creates the release, which we don't want since we have multiple jobs.

Reply:
thanks